src/config.js

```diff
@@ -2,17 +2,12 @@ import $ from 'jquery'
 
 import { setLogDebug } from 'ovirt-ui-components'
 
-// For Development
-// const CONFIG_URL = '/userportal.config'
-
-// TODO: configure path automatically
-// For Production
 const CONFIG_URL = '/ovirt-engine/web-ui/userportal.config'
 
 const AppConfiguration = {
   debug: true,
-  applicationContext: '/ovirt-engine',
-  applicationURL: '/ovirt-engine/web-ui',
+  applicationContext: '',
```

Comment:
So for production, there MUST be the config file available.
A little bit unusual - working defaults would be better.
What about adding new (additional) config file ovirt-web-ui.devel.config which will be loaded when userportal.config (will be renamed later) is missing?